extensions/http-curl/processors/InvokeHTTP.cpp

```diff
@@ -95,9 +95,12 @@ core::Property InvokeHTTP::ContentType("Content-type", "The Content-Type to spec
                                        "POST or PATCH. In the case of an empty value after evaluating an expression language expression, "
                                        "Content-Type defaults to",
                                        "application/octet-stream");
-core::Property InvokeHTTP::SendBody("send-message-body", "If true, sends the HTTP message body on POST/PUT/PATCH requests (default).  "
-                                    "If false, suppresses the message body and content-type header for these requests.",
-                                    "true");
+core::Property InvokeHTTP::SendBody(
+    core::PropertyBuilder::createProperty("send-message-body", "Send Body")
```

Comment:
NiFi has a similar property called "Send Message Body". I would rename this to match.

Reply:
Updated in e27db873853dc331d7c9c20462eb9038f537a6a3